src/main/java/net/minestom/server/event/instance/InstanceRegisteredEvent.java

```diff
@@ -0,0 +1,19 @@
+package net.minestom.server.event.instance;
+
+import net.minestom.server.event.trait.InstanceEvent;
+import net.minestom.server.instance.Instance;
+import org.jetbrains.annotations.NotNull;
+
+public class InstanceRegisteredEvent implements InstanceEvent {
```

Comment:
This is not the right PR to introduce a new event

Reply:
Oh, sorry
My bad